package.json

```diff
@@ -18,14 +18,15 @@
   "types": "lib/index.d.ts",
   "scripts": {
     "build": "npx tsc",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "postinstall": "npm run build"
```

Comment:
this shouldn't be needed i think, prepublish already does this iirc

Reply:
Ok